ros/src/computing/perception/prediction/motion_predictor/packages/naive_motion_predict/include/naive_motion_predict.h

```diff
@@ -41,7 +41,7 @@ class NaiveMotionPredict
   ros::NodeHandle private_nh_;
 
   // max prediction score
-  const double MAX_PREDICTION_SCORE_;
+  double MAX_PREDICTION_SCORE_;
```

Comment:
This change doesn't seem necessary... We still aren't changing this value so it can stay const.